src/components/SEO/default.tsx

```diff
@@ -20,7 +20,7 @@ const SEOHeader = ({ router }: SEOHeaderProps) => {
         site_name: 'IQ Oraqles',
         images: [
           {
-            url: 'https://iq.wiki/images/og-image-default.png',
+            url: '/images/og-image-default.png',
```

Comment:
I think the url can be [https://oraqles-ui.vercel.app/images/og-image-default.png](https://oraqles-ui.vercel.app/images/og-image-default.png) for now.

Reply:
ohh okay